clusters.py

```diff
@@ -259,25 +263,25 @@ def ConvertSpaces(column_names: List) -> List:
   column_names = [s.replace(' ', '_') for s in column_names]
   return column_names
 
-def ConvertToNumerical(sd: List[dataclasses.dataclass],
-                  field_list: List[str],
-                  type=np.float32) -> np.ndarray:
+def ConvertToNumerical_temp(rows_of_data: List, 
+                       field_list: List[str], 
+                       type = np.float32) -> np.ndarray:
   """
-  Collect the data from each field listed in list and convert the string value
-  to a numerical value
-
+  Converts a list of rows containing data into a numerical NumPy array.
+  
   Args:
-    - sd: A list of Stanford dataclass objects (holding the audiogram data)
-    - field_list: A list of column names to extract
-    - type: The desired data type, should be a NP type or object.
+      rows_of_data (List): A list of rows, where each row is an iterable containing data values.
+      field_list (List[str]): A list of field names or labels.
+      type (data type, optional): The data type to use for the NumPy array. Default is np.float32.
 
   Returns:
-    - numpy array of desired datatype (NP type or object)
+      np.ndarray: A numerical NumPy array containing the converted data.
   """
-  data = np.zeros((len(sd), len(field_list)), dtype=type)
-  for i, r in enumerate(sd):
-    for j, f in enumerate(field_list):
-      d = r.__getattribute__(f)
+
+  data = np.zeros((len(rows_of_data), len(field_list)), dtype=type)
```

Comment:
I don't think field_list is still required since you can also get the width of the data from the first row of rows_of_data, right?

Reply:
I am passing only from the 2nd row in the rows imported from the spreadsheet. I thought that this function could be more generalised and hence retained the field list variable.